ui/packages/atlasmap/src/UI/Canvas/Arc.tsx

```diff
@@ -100,7 +80,7 @@ export const Arc: FunctionComponent<IArcProps> = ({
         className={css(props.onClick && styles.clickable)}
         {...props}
       />
-      <ArcEdge
+      {/* <ArcEdge
```

Comment:
If this is something you'd like to keep then you should create an issue and reference it in the commented out code.  Otherwise we'll forget why we commented it out.

Reply:
Thanks, it's save to throw away. I forgot I had it commented.